tss/manager/cmd.go

```diff
@@ -43,20 +43,22 @@ func run(cmd *cobra.Command) error {
 	config := common.GetConfigFromCmd(cmd)
 	log.Info("config info print", "MissSignedNumber", config.MissSignedNumber)
 	log.Info("l1 start block number", "block number", config.L1StartBlockNumber)
-	if len(config.Manager.PrivateKey) == 0 {
-		return errors.New("need to config private key")
-	}
-	wsServer, err := server.NewWSServer(config.Manager.WsAddr)
+	l1StartBlockNumber, err := strconv.ParseUint(
+		config.L1StartBlockNumber, 10, 32,
+	)
 	if err != nil {
 		return err
 	}
+
 	managerStore, err := store.NewStorage(config.Manager.DBDir)
 	if err != nil {
 		return err
 	}
-	l1StartBlockNumber, err := strconv.ParseUint(
-		config.L1StartBlockNumber, 10, 32,
-	)
+	queryService := l1chain.NewQueryService(config.L1Url, config.TssGroupContractAddress, config.L1ConfirmBlocks, managerStore)
```

Comment:
I think it would be better that `NewQueryService` return a pointer and an error.